README.md

```diff
@@ -88,8 +98,13 @@ directory could be mounted as a data volume. e.g.:
       --ipc=host \
       --user=root \
       --privileged \
-      -v /:/host:Z \
-      -v /var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket
-      -it kubevirt/libvirt:latest
+      --volume=/:/host:Z \
+      --volume=/var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket \
+      --volume=/dev:/host-dev \
+      --volume=/var/run/libvirt:/var/run/libvirt \
+      --volume=/sys:/host-sys \
```

Comment:
Here again I'd reorder things, and specifically put `/var/run/dbus/system_bus_socket` right before `/var/run/libvirt`.

All other comments also apply, of course.